Templates/QueueTrigger-Powershell/run.ps1

```diff
@@ -1,3 +1,4 @@
-$in = Get-Content $Env:input
-
-[Console]::WriteLine("Powershell script processed queue message '$in'")
\ No newline at end of file
+$in = Get-Content $triggerInput
+$json = $in | ConvertFrom-Json
+$entity = [string]::Format('{{ "Status": 0, "Title": "PowerShell Table Entity for message {0}" }}', $json.id)

```

Comment:
You need to rebase your branch against latest. A lot of the templates have been simplified - e.g. this template no longer writes to an output table